.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -142,8 +142,16 @@ object WPComPlugins_EditorToolKit : BuildType({
 				# 3. Check if the current build has changed, and if so, tag it for release.
 				# Note: we exclude asset changes because we only really care if the build files (JS/CSS) change. That file is basically just metadata.
 				if ! diff -rq --exclude="*.asset.php" ./editing-toolkit-plugin/ ./current-etk-release/ ; then
-					echo "The build is different from the most current release build. Therefore, this can be tagged as a new release build."
-					curl -X POST -H "Content-Type: text/plain" --data "etk-release-build" -u "%system.teamcity.auth.userId%:%system.teamcity.auth.password%" %teamcity.serverUrl%/httpAuth/app/rest/builds/id:%teamcity.build.id%/tags/
+					echo "The build is different from the last release build. Therefore, this can be tagged as a release build."
+					tag_response=`curl -s -X POST -H "Content-Type: text/plain" --data "etk-release-build" -u "%system.teamcity.auth.userId%:%system.teamcity.auth.password%" %teamcity.serverUrl%/httpAuth/app/rest/builds/id:%teamcity.build.id%/tags/`
+					echo -e "Build tagging status: ${'$'}tag_response\n"
```

Comment:
Is this to escape `$` so that this is not interpreted by Kotlin as variable interpolation? If that's the case, why isn't it possible to escape with `\`?

Reply:
Right on! Unfortunately, [escaping is not supported](https://kotlinlang.org/docs/basic-types.html#string-literals) in multi-line/raw strings in Kotlin:

> A raw string is delimited by a triple quote (""" ), contains **no escaping** and can contain newlines and any other characters:



`${'$'}` is the recommended way to accomplish this for bash scripts in the TeamCity docs as well, and it's definitely really ugly.